language-plutus-core/test/type-errors/wrapType.plc

```diff
@@ -1,6 +0,0 @@
-(program 0.1.0
```

Comment:
If these are not type errors now, please witness that by moving them to the folder for succeeding tests.

Reply:
This is just an invalid test. It fails `checkFile` because it is not normalized

Reply:
If there's a bug in the test, why not just fix it instead of throwing it away? You could easily fix this in the manner we've discussed before.

Reply:
For one, the test should never have been added. It was a bogus test. 

The fact that the test suite was in such a shoddy state and the fact that it didn't prevent us from doing the right thing is an argument that the test suite is *not* helping development.

Reply:
Your reaction to "this didn't work" seems to be "throw it out" instead of "fix it". I don't understand why, given that it's very easy to fix, and we want to be gradually building up a good test suite.

Reply:
If the test suite is not helping development, then we must improve the test suite. It is definitely possible to have a helpful test suite and a good testsuite makes a big difference.

Reply:
Also, I've realised I was even more confused: this is a good test case that *remains* a good test case. It's in a directory of test cases specifically for errors. So it's supposed to be there witnessing the fact that you can't currently use normalized types in this way. The current error is wrong, it should get a normalization error, but it absolutely belongs in the test suite.

Reply:
I'm certainly willing to improve the test suite, possibly with property tests. 

> Your reaction to "this didn't work" seems to be "throw it out" instead of "fix it". I don't understand why, given that it's very easy to fix

If there is nothing to prevent us from adding future shoddy test cases from being added, it is not much good.

Reply:
> If there is nothing to prevent us from adding future shoddy test cases from being added, it is not much good.

What isn't? Having a good test case is good, so if we have a not good test case, let's fix it and make sure we don't make the mistake again. In this case I think all we need to do is make sure we run the type checking tests through the normalization checker first, which is easy.

And I reiterate: this test is not bad, it is deliberately testing a type error, which it is doing correctly (although it should be reported as a normalization error).